kanidmd/src/lib/be/dbvalue.rs

```diff
@@ -70,41 +87,64 @@ pub struct DbCredV1 {
 
 #[derive(Serialize, Deserialize, Debug)]
 pub struct DbValueCredV1 {
-    pub t: String,
-    pub d: DbCredV1,
+    #[serde(rename = "t")]
+    pub tag: String,
+    #[serde(rename = "d")]
+    pub data: DbCredV1,
 }
 
 #[derive(Serialize, Deserialize, Debug)]
 pub struct DbValueTaggedStringV1 {
-    pub t: String,
-    pub d: String,
+    #[serde(rename = "t")]
+    pub tag: String,
+    #[serde(rename = "d")]
+    pub data: String,
 }
 
 #[derive(Serialize, Deserialize, Debug)]
 pub struct DbValueEmailAddressV1 {
-    pub d: String,
+    #[serde(rename = "d")]
+    pub email_addr: String,
 }
 
 #[derive(Serialize, Deserialize, Debug)]
 pub enum DbValueV1 {
-    U8(String),
-    I8(String),
-    N8(String),
-    UU(Uuid),
-    BO(bool),
-    SY(usize),
-    IN(usize),
-    RF(Uuid),
-    JF(String),
-    CR(DbValueCredV1),
-    RU(String),
-    SK(DbValueTaggedStringV1),
-    SP(String, String),
-    UI(u32),
-    CI(DbCidV1),
-    NU(String),
-    DT(String),
-    EM(DbValueEmailAddressV1),
+    #[serde(rename = "U8")]
+    Utf8(String),
+    #[serde(rename = "I8")]
+    Iutf8(String),
+    #[serde(rename = "N8")]
+    Iname(String),
+    #[serde(rename = "UU")]
+    Uuid(Uuid),
+    #[serde(rename = "BO")]
+    Bool(bool),
+    #[serde(rename = "SY")]
+    SynType(usize),
+    #[serde(rename = "IN")]
+    IdxType(usize),
+    #[serde(rename = "RF")]
+    Refer(Uuid),
+    #[serde(rename = "JF")]
+    JsonFilter(String),
+    #[serde(rename = "CR")]
+    Cred(DbValueCredV1),
+    #[serde(rename = "RU")]
+    RadiusCred(String),
```

Comment:
I'm about to rename this to "SecretValue" in my oauth2 pr, so change RadiusCred to SecretValue, but leave it as RU on the db format.